columnflow/tasks/yields.py

```diff
@@ -222,7 +230,7 @@ def run(self):
 
                     # format yields into strings
                     yields_str[category].append((value / norm_factor).str(
-                        combine_uncs="all",
+                        # combine_uncs="all",  # TODO
```

Comment:
intended?

Reply:
Yes, there was an error at that line and I wasn't motivated enough to find out what the issue is.

Reply:
:+1: Could you open an issue referring to this line, just so that we can follow up.

Reply:
Opened! #229